test/testsuite/vector.jl

```diff
@@ -21,6 +21,13 @@
     # we don't shrink buffers yet... TODO shrink them... or should we?
     @test length(GPUArrays.buffer(x)) == 5
 
+    a = Float32[0, 1, 2]
+    x = AT(a)
+    append!(x, [3, 4])
+    @test length(x) == 5
+    @test x[4] == 3
+    @test x[5] == 4
+
```

Comment:
Just a heads up. The vector tests seem to be disabled currently: https://github.com/JuliaGPU/GPUArrays.jl/blob/228fb9e8f878333cc0eda8678c8e7c2d6b5a6df2/test/testsuite.jl#L91